app/src/androidTest/java/org/liberty/android/fantastischmemo/test/db/SettingTest.java

```diff
@@ -105,6 +105,15 @@ public void testAnswerAudio() throws Exception {
         assertEquals("DE", savedSetting.getAnswerAudio());
     }
 
+    @SmallTest
+    @Test
+    public void testHintAudio() throws Exception {
+        Setting setting = getCurrentSetting();
+        setting.setHintAudio("DE");
+        Setting savedSetting = setSetting(setting);
+        assertEquals("DE", savedSetting.getHintAudio());
+    }
```

Comment:
Is the audio sound set when the user adds a hint? Or do we have to set it after? What are the steps for setting a hint and then having the audio set?

Reply:
The hint audio by default is set to "English(US)", so it is always on from the start. However, the user also has the option to set the hint audio to "Disabled" in the settings of the deck, which turns off the TTS for hint. The same thing can be done to question and answer.